source/OptimaJet.DWKit.StarterApplication/Models/ProductActions.cs

```diff
@@ -6,7 +6,22 @@ namespace OptimaJet.DWKit.StarterApplication.Models
 {
     public class ProductActions : Identifiable<Guid>
     {
-        [Attr("types")]
-        public List<string> Types { get; set; }
+        [Attr("actions")]
+        public List<string> Actions { get; set; }
+    }
+
+    public class ProductActionRuns : Identifiable<Guid>
```

Comment:
`ProductActionRuns` tells me that these are things that have been run.

Maybe `AvailableProductsForAction`?

Reply:
Changed to ProductActionRunRequest